src/api/routes/projects.js

```diff
@@ -1,21 +1,29 @@
 const ProjectsService = require('../route-services/projects');
 
+const { expressAuthenticationOnlyMiddleware } = require('../../utils/authMiddlewares');
+
 const projectsService = new ProjectsService();
 
 module.exports = {
-  'projects#update': (req, res, next) => {
-    projectsService.updateProject(req.params.projectUuid, req.body)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-  'projects#delete': (req, res, next) => {
-    projectsService.deleteProject(req.params.projectUuid)
-      .then((data) => res.json(data))
-      .catch(next);
-  },
-
-  'projects#get': (req, res, next) => {
-    projectsService.getProjects().then((response) => res.json(response)).catch(next);
-  },
+  'projects#update': [
+    expressAuthenticationOnlyMiddleware,
```

Comment:
I think that we are using the wring middleware here and need also to check for authorization: `expressAuthorizationMiddleware`

Reply:
This requires the capability to check for Auth by projects. I'll implement it together with this ticket,

Reply:
Implemented in `authMiddleware`